File: hospital/SergioDeIscar-hospital-java/SergioDeIscar-hospital-java/src/main/java/org/example/exceptions/HospitalExceptions.java
package org.example.exceptions;

public abstract class HospitalExceptions extends RuntimeException {
    public HospitalExceptions(String message) {
        super(message);
    }

    public static final class HospitalFullException extends HospitalExceptions {
        public HospitalFullException() {
            super("Error -> Hospital lleno.");
        }
    }

    public static final class DniNotValidException extends HospitalExceptions {
        public DniNotValidException() {
            super("Error -> DNI no válido.");
        }
    }
}

File: hospital/SergioDeIscar-hospital-kotlin/SergioDeIscar-hospital-kotlin/src/main/kotlin/exceptions/HospitalExceptions.kt
package exceptions

sealed class HospitalExceptions(message: String): RuntimeException(message)
class HospitalFullException: HospitalExceptions("Error -> Hospital lleno.")
// class PacienteNotFoundException: HospitalExceptions("Error -> Paciente no encontrado.")
class DniNotValidException: HospitalExceptions("Error -> DNI no válido.")